web/controllers/user_controller.ex

```diff
@@ -28,9 +38,15 @@ defmodule CoursePlanner.UserController do
       {:ok, user} ->
         Users.notify_user(user, current_user, :user_modified, user_show_url(user))
 
-        conn
-        |> put_flash(:info, "User updated successfully.")
-        |> redirect(to: dashboard_path(conn, :show))
+        if conn.assigns.current_user.role === "Coordinator" do
```

Comment:
what do you think about checking `conn.assigns.current_user.id == id` instead?
my idea is that coordinators can see the message `"Your profile has been updated successfully."` if they edit themselves